test/src/api/endpoints/test-work.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable */
+
+process.env.NODE_ENV = 'test';
+
+import app from '../../../../src/server/app';
+/// Import the dependencies for testing
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import orm from '../../../bookbrainz-data';
+// Configure chai
+chai.use(chaiHttp);
+chai.should();
+
+const {
+	Alias, AliasSet, Annotation, Disambiguation, Editor, EditorType, Entity, Gender,
+	IdentifierSet, RelationshipSet, Revision, Work, bookshelf, util
+} = orm;
+
+const genderData = {
+	id: 1,
+	name: 'test'
+};
+const editorTypeData = {
+	id: 1,
+	label: 'test_type'
+};
+const editorData = {
+	genderId: 1,
+	id: 1,
+	name: 'bob',
+	typeId: 1
+};
+const setData = {id: 1};
+
+const aliasData = {
+	id: 1,
+	name: 'work name',
+	sortName: 'Work sort name'
+};
+
+const revisionAttribs = {
+	authorId: 1,
+	id: 1
+};
+const aBBID = '13ab71b4-908f-4c72-88a9-4b94a56c7d3e';
```

Comment:
You should make this a random uuid. I can see a couple of potential issues otherwise that could be avoided.
I suggest following what is done in bookbrainz-data-js and installing the 'faker' package, then using that for bbids, names, etc.
https://github.com/bookbrainz/bookbrainz-data-js/search?q=faker&unscoped_q=faker